File: 백준/Bronze/10950. A＋B － 3/A＋B － 3.py
import math
import heapq
from collections import deque
from itertools import permutations, combinations
import sys
input = sys.stdin.readline


if __name__ == "__main__":
    T = int(input())

    for i in range(T):
        a, b = map(int,input().split())
        print(a+b)

File: 백준/Bronze/10950. A＋B － 3/A＋B － 3.js
var input = require('fs').readFileSync('/dev/stdin').toString().trim().split(/\s+/).map(Number);

let N = input[0];
let idx = 1;
let out = [];

for(let i = 0; i < N; i++){
    let a = input[idx++];
    let b = input[idx++];
    out.push(a + b);
}

console.log(out.join('\n'));